app/models/flatirons/saas/subscription.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Flatirons::Saas
+  # Subscription Active Record
+  #
+  # == Examples
+  #
+  #   # get all subscriptions
+  #   Flatirons::Saas:Subscription.all
+  #
```

Comment:
Same

Reply:
it is used to generate the documentation to our library.